src/test/java/guitests/StartupBoardLauncherTest.java

```diff
@@ -0,0 +1,53 @@
+package guitests;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.loadui.testfx.utils.FXTestUtils;
+import prefs.Preferences;
+import ui.BoardAutoCreator;
+import ui.TestController;
+import ui.UI;
+import ui.issuepanel.PanelControl;
+import static guitests.BoardAutoCreatorTest.verifyBoard;
+
+import java.util.Optional;
+
+import static org.junit.Assert.assertEquals;
+import static ui.BoardAutoCreator.SAMPLE_BOARD;
+
+public class StartupBoardLauncherTest extends UITest{
+
+    private PanelControl panelControl;
+    private Preferences testPref;
+
+    @Override
+    public void launchApp(){
+        FXTestUtils.launchApp(TestUI.class, "--startupboard=true");
+    }
+
+    @Before
+    public void init(){
+        UI ui = TestController.getUI();
+        panelControl = ui.getPanelControl();
+        testPref = UI.prefs;
+    }
+
+    @Test
+    public void testSampleBoardOnLaunch(){
+        login("dummy", "dummy", "test", "test");
+        waitUntilNodeAppears(panelControl.getPanel(0));
+        assertEquals(panelControl.getPanelCount(), BoardAutoCreator.getSamplePanelDetails().size());
+        assertEquals(panelControl.getCurrentlySelectedPanel(), Optional.of(0));
+        assertEquals(panelControl.getNumberOfSavedBoards(), 1);
+
+        assertEquals(testPref.getAllBoardNames().get(0), SAMPLE_BOARD);
+        verifyBoard(panelControl.getCurrentPanelInfos(), BoardAutoCreator.getSamplePanelDetails());
+    }
```

Comment:
Both are needed as that one checks whether the board was created correctly through the `Auto Create` menu. This one is for new users.